docs/Automation.md

```diff
@@ -1,183 +1,205 @@
-These objects are detailed under each command:
-
-### Command Details
-#### Start Command
-The Start command initializes the AxeWindows engine for automation.
-It is required for the session to begin, and each session requires a
-corresponding call to the Stop command. A given test process is allowed only one
-active session at a time, and subsequent calls to Start will fail.
-
-##### Inputs
-
-###### .NET
-
-The **StartCommand.Execute** method accepts the following parameters:
+### How to run an automated scan
 
-**Name** | **Type** | **Description**
----|---|---
-primaryConfig | `Dictionary<string, string>` | The set of Tier 2 parameters to apply during this session. See [Defined Constants](#defined-constants) for specific parameters.
-configFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Can be `string.Empty`. Must point to a valid JSON-serialized file if non-empty.
+1. Create a `Config` object using `Config.Builder`.
 
-###### PowerShell
+        // Create config to specifically target a process
+        var myConfigBuilder = Config.Builder.ForProcessId(1234);
 
-The **Start-AxeWindows** cmdlet accepts the following optional
-parameters:
+        // Optional: configure to create an A11yTest file
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
 
-**Name** | **Type** | **Description**
----|---|---
-OutputFile | `string`  | The location where output from this session will be written. The file extension can (and should) be omitted. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-OutputFileFormat | `string` | The output file format to generate. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-TargetProcessId | `string` | The process id, expressed as a string, that will be scanned. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-ConfigFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Must point to a valid JSON-serialized file if specified.
+        // Optional: configure to output the file to a specific directory (otherwise, current directory will be used)
+        myConfigBuilder.WithOutputDirectory(".\test-directory");
+
+        // Ready to use config
+        var myConfig = myConfigBuilder.build();
+
+2. Create a `Scanner` object using the `ScannerFactory` object with the `Config`.
+
+        // Create scanner using myConfig
+        var scanner = ScannerFactory.CreateScanner(myConfig);
+
+3. Call  the `Scan` method on the `Scanner` object.
+
+        var scanResults;
+        try
+        {
+            scanResults = scanner.Scan();
+        }
+        catch(AxeWindowsAutomationException e)
+        {
+            // Get the message from an exception, if one is thrown.
+            var errorMessage = e.InnerException != null ? e.InnerException.Message : e.Message;
```

Comment:
There's at least 1 place (Line 23 of TargetElementLocator.cs) where this might return a less-than-helpful message. Would it make sense to just ToString() the whole Exception?
```suggestion
            Console.WriteLine(e.ToString());
```